Client/Frontend/Browser/SearchViewController.swift

```diff
@@ -164,6 +167,28 @@ class SearchViewController: SiteTableViewController,
         }
     }
 
+    private func loadFirefoxSuggestions() {
+        guard featureFlags.isFeatureEnabled(.firefoxSuggestFeature, checking: .userOnly) else {
```

Comment:
Will do! I think restricting it to the `developer` channel for now makes sense—I'll give it a try, and let you know if I run into issues!

Reply:
Alrighty, I _think_ I got the plumbing for the Nimbus feature set up—it's the second commit in the stack, plus a change to `AppSettingsTableViewController` in the third commit 😊 Here's how it works:

* The hidden Firefox Suggest debug menu is only available on the developer channel—its visibility is controlled by whether the Nimbus feature variable is set for that channel (IOW, we check the flag with `.buildOnly`). So beta and release users won't see the Suggest debug menu at all.
* If you're on the developer channel, you'll see the debug menu, and you'll _also_ need to flip the switch in the menu to turn on Suggest (and trigger an ingestion, if you're in the Simulator, or don't want to wait for the background task to run), before you see suggestions. IOW, when we ingest, query, and show suggestions, we check the feature flag using `.buildAndUser`.

I added the additional switch in the debug menu for a couple of reasons:

* I wanted to avoid showing suggestions in the awesomebar until we have new Search settings UI to control it.
* I wanted to provide a way to override the default value of the Nimbus variable—for testing, or if you're not part of the eligible Suggest population, but want to try it, anyway.

How does that sound?